module/input/NatNet/src/NatNet.cpp

```diff
@@ -425,11 +436,12 @@ namespace module::input {
         // Get our packet
         const Packet& packet = *reinterpret_cast<const Packet*>(input.data());
 
-        // std::stringstream ss;
-        // for (const auto& c : input) {
-        //     ss << std::setw(2) << std::setfill('0') << std::hex << int(c);
-        // }
-        // NUClear::log(ss.str());
+        // This isn't working yet for some reason, no file is being created
```

Comment:
This should be a todo so you don't forget it.
But also get it working
You might need a makedirs call here since if you're making this in a folder that doesn't exist it won't work